src/components/NewBook/index.js

```diff
@@ -9,42 +9,45 @@ import './newBook.css';
 
 const NewBook = () => {
```

Comment:
- [ ] When I change tabs from books to categories to books, the component books rerenders again and fetches the books from the API again. This is not the intended behavior and styling should not modify it. Let's try to fix this. I will give you a tip: you need to check if you have fetch already the books when you `dispatch(loadBooks())` :wink: 

![bookstore-rerender](https://user-images.githubusercontent.com/79114233/145503569-4f905585-f458-411d-934d-82384193c3d1.gif)

Reply:
Thanks! I didn't notice this, I just fixed this by using a simple if statement asking if the document is ready or not...